callstack/repack
# fix: get commands options and description from new package

### Summary

Fixes https://github.com/callstack/repack/issues/473. With React Native 0.73, [we've migrated](https://github.com/facebook/react-native/pull/38795) `start` and `bundle` command to Core. So right now when resolving path to CLI and trying to require these commands from CLI, script will fail. In this PR I changed the `require.resolve` to look into new package, where these commands are living right now.

### Test plan

1. Create new React Native app with `npx react-native@latest init app`
2. Init Re.Pack with `npx @callstack/reapck-init`
3. Apply changes from this PR
4. `npx react-native config` should be executed correctly.
